meta-iotqa/lib/oeqa/runtime/multimedia/vaapi/test_vaapi_encode_decode.py

```
@@ -0,0 +1,198 @@
+                'files',
+                VAAPITest.h264_encoded_file),
+            VAAPITest.h264_encoded_file_target)
+        '''
+        self.target.copy_to(
+            os.path.join(
+                os.path.dirname(__file__),
+                'files',
+                VAAPITest.mpeg2_encoded_file),
+            VAAPITest.mpeg2_encoded_file_target)
+
+        self.target.copy_to(
+            os.path.join(
+                os.path.dirname(__file__),
+                'files',
+                VAAPITest.vc1_encoded_file),
+            VAAPITest.vc1_encoded_file_target)
+        '''
+        self.target.copy_to(
+            os.path.join(
+                os.path.dirname(__file__),
+                'files',
+                VAAPITest.mjpeg_encoded_file),
+            VAAPITest.mjpeg_encoded_file_target)
+
+        self.target.copy_to(
+            os.path.join(
+                os.path.dirname(__file__),
+                'files',
+                VAAPITest.vp8_encoded_file),
+            VAAPITest.vp8_encoded_file_target)
+
+        # get supported vaapi encoders and decoders
+
+        (status, output) = self.target.run(" gst-inspect-1.0 | grep vaapi")
+        output = output.lower()
+        self.available_vaapi_elements = output
+
+        '''
+        self.target.copy_to(
+            os.path.join(
+                os.path.dirname(__file__),
+                'files',
+                VAAPITest.vp9_encoded_file),
+            VAAPITest.vp9_encoded_file_target)
+        '''
+    # Decrey H.264 encoded file using VAAPI
+
+    def test_gst_vaapi_can_decode_h264(self):
+        if "vaapih264dec" in self.available_vaapi_elements :
+            (status, output) = self.target.run(
+                'gst-launch-1.0 filesrc location= %s ! qtdemux ! vaapih264dec ! vaapisink' % VAAPITest.h264_encoded_file_target)
+            self.assertEqual(status, 0, msg="Error messages: %s" % output)
+        else :
+            print ("vaapi h264 decode not supported ")
```

Or should the test fail instead of printing a message?